src/js/components/Button/StyledButton.js

```diff
@@ -54,7 +54,12 @@ const padStyle = props => {
 const basicStyle = props => css`
   border: ${props.theme.button.border.width} solid
     ${normalizeColor(
-      props.colorValue || props.theme.button.border.color || 'control',
+      (props.active &&
+        props.primary &&
+        props.theme.button.primary.active.border.color) ||
+        props.colorValue ||
+        props.theme.button.border.color ||
+        'control',
```

Comment:
Since we changed the base.js to default to `control` I don't think we will get to the other conditions on this list, I wonder if it should be the default and documented to `undefined`. thoughts?

Reply:
I think we still need the other conditions because the first set of conditions just checks in the case where a button is `active` and `primary`, otherwise it should do they same series of conditions as before.

Reply:
Simplified this section by creating a `activeButtonStyle` that does this check. It makes more sense for it to live in an `active` function as opposed to the `basicStyle`. Using the mindset we had applied to disabledStyle in that it uses the `activeStyle` from utils and then adds on any button specific stlying.